src/main/kotlin/com/pubnub/api/subscribe/eventengine/effect/RemoteActionManagedEffect.kt

```diff
@@ -1,28 +0,0 @@
-package com.pubnub.api.subscribe.eventengine.effect
-
-import com.pubnub.api.endpoints.remoteaction.RemoteAction
-import com.pubnub.api.eventengine.ManagedEffect
-import com.pubnub.api.models.consumer.PNStatus
-
-class RemoteActionManagedEffect<T>(
```

Comment:
What's the reason for removal? Together with `DelayedManagedEffect` it seems to be powerful enough to fulfill our needs and on the other hand the implementation of both is concise and focused

Reply:
Yes, this code was concise. Removal of this class was consequence of moving Effects to separate classes. As discussed we will try to extract some common code for Effect. Maybe at that time code from this class will be used.